packages/kit-headless/project.json

```diff
@@ -88,6 +88,21 @@
         "project": ["logic"]
       }
     },
+    "setup-chrome-113": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "setup:chrome:113"
+      }
+    },
+    "e2e-chrome-113": {
+      "executor": "@nx/playwright:playwright",
+      "outputs": ["{workspaceRoot}/dist/.playwright/packages/kit-headless"],
+      "dependsOn": ["setup-chrome-113"],
```

Comment:
Setup task always runs before the actual execution but bypasses if the folder already exists to improve performance.